hyprpm/src/core/PluginManager.cpp

```diff
@@ -847,17 +868,29 @@ ePluginLoadStateReturn CPluginManager::ensurePluginsLoadState() {
             if (std::find_if(loadedPlugins.begin(), loadedPlugins.end(), [&](const auto& other) { return other == p.name; }) != loadedPlugins.end())
                 continue;
 
-            loadUnloadPlugin(HYPRPMPATH + repoForName(p.name) + "/" + p.filename, true);
-            std::println("{}", successString("Loaded {}", p.name));
+            if (!loadUnloadPlugin(HYPRPMPATH + repoForName(p.name) + "/" + p.filename, true)) {
+                std::println("{}", infoString("{} will be loaded after restarting Hyprland", p.name));
+                hyprlandVersionMismatch = true;
+            } else {
```

Comment:
ahem no {} around short ifs

Reply:
it won't compile if i get rid of the braces like so:
```
if (!loadUnloadPlugin(HYPRPMPATH + repoForName(p.name) + "/" + p.filename, true))
    std::println("{}", infoString("{} will be loaded after restarting Hyprland", p.name));
    hyprlandVersionMismatch = true;
else
    std::println("{}", successString("Loaded {}", p.name));
```
so i kept them. clang does not know how to deal with this and frankly it looks like python. ill remove them from the lower part tho

Reply:
just else